marimbabot_audio/src/audio_from_lilypond.py

```diff
@@ -76,19 +89,59 @@ def callback_vision_results(data: String, audio_publisher):
     # publish audio file to audio node (sound_play package)
     audio_publisher.publish(sound_request)
 
-# create audio from lilypond publisher
-# send audio to audio node (sound_play package)
-def listener():
+    return audio_filename
+
+"""
+Action server for audio generation from lilypond string
+"""
+class AudioFromLilypondActionServer(object):
```

Comment:
Idk if I would name it action server. You also don't name the classes of other nodes `XYZSubscriber`. It is not wrong, but the action server is just an interface for this code.

Reply:
done